scan.go

```diff
@@ -577,6 +576,7 @@ func recvResults(w http.ResponseWriter, r *http.Request) {
 		gaugeTotal.Set(float64(results.Total))
 		gaugeLatest.Set(float64(results.Latest))
 		gaugeNew.Set(float64(results.New))
+		gaugeSubmission.Set(float64(time.Now().Unix()))
```

Comment:
I think this should already be updated, right? `saveSubmission` (called a few lines up) also updates this gauge. Is it not updating?

Reply:
You're right and it does seem to be working. My bad.